libs/cart/driver/magento/src/queries/set-guest-email.ts

```diff
@@ -3,17 +3,20 @@ import gql from 'graphql-tag';
 
 import { daffBuildFragmentNameSpread, daffBuildFragmentDefinition } from '@daffodil/core/graphql';
 
+import { cartFragment } from './fragments/cart';
+
 export const setGuestEmail = (extraCartFragments: DocumentNode[] = []) => gql`
   mutation SetGuestEmail($cartId: String!, $email: String!) {
     setGuestEmailOnCart(input: {
       cart_id: $cartId,
       email: $email
     }) {
       cart {
-        email
+        ...cart
```

Comment:
I don't think this change is needed. I don't even think this query is actually used anywhere anyway.

Reply:
I just removed this since it wasn't used